src/main/java/de/metas/ui/web/order/sales/purchasePlanning/view/PurchaseRow.java

```diff
@@ -162,6 +170,7 @@ private PurchaseRow(
 		this.rowType = rowType;
 		this.product = product;
 		this.vendorBPartner = vendorBPartner;
+		this.availableQty = availableQty;
```

Comment:
why not calling it qtyAvailable? ...it would respect the same naming conventions like qtyToDeliver, qtyToPurchase.